common/src/main/scala/uk/ac/wellcome/lib/elasticsearch/mappings.scala

```diff
@@ -0,0 +1,31 @@
+package uk.ac.wellcome.lib.elasticsearch
+
+import com.sksamuel.elastic4s._
+import com.sksamuel.elastic4s.ElasticDsl._
+import com.sksamuel.elastic4s.mappings.FieldType._
+import com.sksamuel.elastic4s.analyzers._
+
+// TODO: This should have a test before any more code is written.
```

Comment:
Sorry ... should have written the tests first. I wasn't sure if this was even possible for our purposes with the elastic4s library, but it seems it is.